backend/resources/akvo/lumen/migrations/tenant_manager/004-remove-plan.up.sql

```diff
@@ -0,0 +1,4 @@
+DROP INDEX plan_tenant_ends_key;
+ALTER TABLE plan DROP CONSTRAINT plan_pkey;
+DROP TABLE IF EXISTS plan;
```

Comment:
`DROP TABLE` support `CASCADE`, so you don't need to delete child objects manually.
https://www.postgresql.org/docs/11/sql-droptable.html

Reply:
keeping just 
```
DROP TABLE IF EXISTS plan;
DROP TYPE IF EXISTS tier;


```